platforms/jvm/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/JavaCompileProblemsIntegrationTest.groovy

```diff
@@ -229,14 +230,14 @@ class JavaCompileProblemsIntegrationTest extends AbstractIntegrationSpec {
     void assertProblem(
         ReceivedProblem problem,
         String severity,
+        boolean expectPreciseLocation = true,
         @ClosureParams(
             value = SimpleType,
             options = [
                 "java.lang.String",
                 "java.lang.String"
             ]
-        )
-        Closure extraChecks = null
+        ) Closure extraChecks = null
```

Comment:
💭
Could we make this `Closure extraChecks = { }`?

This would avoid the check for `null` a few lines down.

I know it is not new code but I couldn't get my eyes of it :)